plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstance.java

```diff
@@ -145,6 +145,7 @@ public MachineRuntimeInfoImpl getRuntime() {
             try {
                 final ContainerInfo containerInfo = docker.inspectContainer(container);
                 machineRuntime = new MachineRuntimeInfoImpl(dockerMachineFactory.createMetadata(containerInfo,
+                                                                                                null,
```

Comment:
As discussed on Slack I've cleaned-up `LocalDockerInstanceRuntimeInfo` class and made it easier to read.